src/components/GraphFilter/__tests__/GraphFind.test.tsx

```diff
@@ -67,21 +67,33 @@ describe('Parse find value test', () => {
     // @ts-ignore
     expect(instance.parseFindValue('outside')).toEqual('node[isOutside]');
     // @ts-ignore
-    expect(instance.parseFindValue('unused')).toEqual('node[isUnused]');
+    expect(instance.parseFindValue('outside')).toEqual('node[isOutside]');
```

Comment:
Minor - I think you can remove this line. I cannot find any difference against the previous line.

Reply:
Good find, actually the second line is supposed to test 'outsider', it was missing the 'r',